charts/linkerd2/values.yaml

```diff
@@ -312,11 +312,11 @@ prometheus:
 
 tracing:
   enabled: false
-  collector:
-    name: linkerd-collector
-    image: omnition/opencensus-collector:0.1.11
-    # resources:
-  jaeger:
-    name: linkerd-jaeger
-    image: jaegertracing/all-in-one:1.17.1
-    # resources:
+  # collector:
+  #   name: linkerd-collector
+  #   image: omnition/opencensus-collector:0.1.11
+  #   resources:
+  # jaeger:
+  #   name: linkerd-jaeger
+  #   image: jaegertracing/all-in-one:1.17.1
+  #   resources:
```

Comment:
These can be removed

Reply:
hmmm? :thinking: We actually comment values that can be allowed for overriden right? just like we did for Prometheus https://github.com/linkerd/linkerd2/blob/main/charts/linkerd2/values.yaml#L268

Reply:
Ah yep I didn't realize that! Thanks